tasks/deploy.yml

```diff
@@ -41,24 +41,6 @@
   become: true
   become_user: "{{ dashboard_user }}"
 
-- name: Copying SAML config files
-  template:
-    src: "{{ item }}"
-    dest: "{{ project_source_new_code_path }}/vendor/simplesamlphp/simplesamlphp/config/{{ item | basename }}"
-    owner: "{{ dashboard_user }}"
-    group: www-data
-    mode: "0640"
-  with_fileglob:
-    - ../templates/saml/config/*.php
-
-- name: Copying SAML metadata files
-  template:
-    src: "saml/metadata/{{ saml2_idp_entry }}/saml20-idp-remote.php"
-    dest: "{{ project_source_new_code_path }}/vendor/simplesamlphp/simplesamlphp/metadata/saml20-idp-remote.php"
-    owner: "{{ dashboard_user }}"
-    group: "{{ dashboard_user }}"
-    mode: "0644"
-
```

Comment:
I'm pretty sure I don't need to use the file module with `state: absent` on these, since they were simply copied into place in the vendor directory, and the target directory won't be there in new deployments. @adborden can you confirm?

Reply:
Actually I'm sure of it.